src/core.js

```diff
@@ -3382,13 +3463,18 @@ export default function Core(rootElement, userSettings, rootInstanceSymbol = fal
     let result = false;
 
     if (row !== void 0 && column !== void 0) {
-      result = instance.view.scrollViewport(new CellCoords(row, column), snapToTop, snapToRight, snapToBottom, snapToLeft);
+      result = instance.view.scrollViewport(
+        fromVisualToRenderableCoords({ row, col: column }), snapToTop, snapToRight, snapToBottom, snapToLeft);
     }
+
     if (typeof row === 'number' && typeof column !== 'number') {
-      result = instance.view.scrollViewportVertically(row, snapToTop, snapToBottom);
+      result = instance.view.scrollViewportVertically(
+        instance.rowIndexMapper.getRenderableFromVisualIndex(row), snapToTop, snapToBottom);
```

Comment:
The new code should look good.

Reply:
Cool 👍 

IMO there are too many calls to Number.isInteger. We could cache the result once for row and column and reuse them. Or since indexMapper will return `undefined` they can be either the integer or `undefined` and there is no reason to check it again? But I should put it in another review

Reply:
My suggestion: https://github.com/handsontable/handsontable/commit/a1610cf4e16a1b4252e3a4d072941342abfc31cd#diff-7eb52b366866677666470e019283c8ea.

Reply:
I was first confused by the names, there are two variables `isRowInteger` one for visible and one for a renderable row so it may be worth to consider naming the second one f.e. `isRenderableRowInteger`. Not that important. Nice work 👌